doc/grants.md

```diff
@@ -42,3 +42,35 @@ should submit one month before the conference.
 
 ### WBI

```

Comment:
```suggestion
# WBI

```

Reply:
@fcerpe as discussed already, I will remove your suggestion since they are not in line with the MD title/paragraph/etc structure